lib/SVF-FE/PAGBuilder.cpp

```diff
@@ -116,6 +116,22 @@ PAG* PAGBuilder::build(SVFModule* svfModule)
         }
     }
 
+    // fill valuToEdgeMap
+    unsigned edge_count = 0;
+    unsigned node_count = 0;
+    for (auto& node : *pag) {
+        node_count++;
+        for (auto edge_it = node.second->OutEdgeBegin(); edge_it != node.second->OutEdgeEnd(); ++edge_it) {
+            auto edge = *edge_it;
+            pag->mapValueToEdge(edge->getValue(), edge);
+            edge_count++;
+        }
+    }
```

Comment:
You can init your map here. It will become simpler.
https://github.com/SVF-tools/SVF/blob/master/lib/SVF-FE/PAGBuilder.cpp#L1437

Reply:
Thanks, will take a look into it.